portfolio/src/main/data_processing/processor.py

```diff
@@ -0,0 +1,77 @@
+""" Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. 
+
+processor.py is used to clean csv's and create data suitable to be
+stored within a database."""
+
+from google.cloud import datastore
+import pandas as pd
+import numpy as np
+import os
+
+def cleanData(raw_file, COLUMNS_OF_INTEREST):
+  raw_file_path = 'raw_data/' + raw_file
+  dataframe = pd.read_csv(raw_file_path, index_col=0)
+  csv_columns = dataframe.columns
+  columns_to_keep = []
+  for csv_column in csv_columns:
+    stripped_csv_column = csv_column.lower()
+    for column_of_interest in COLUMNS_OF_INTEREST:
+        if column_of_interest.lower() in stripped_csv_column:
+          columns_to_keep.append(csv_column)
+          break
+      
+  dataframe = dataframe[columns_to_keep]
+  clean_file_path = 'clean_data/' + raw_file
+  dataframe.to_csv(clean_file_path, index = False)
```

Comment:
Same thing here. This might leave out an index which is a column you want to keep.

Reply:
I'm going to try and see if messing around with the indexes with change the way it operates.